pom.xml

```diff
@@ -1112,6 +1138,13 @@
         <extensions>true</extensions>
         <inherited>true</inherited>
       </plugin>
+      <plugin>
```

Comment:
I was thinking that it might be good to create a `build-tools` module as a compile-only, optional dependency, with checkstyle and formatter styles in `src/main/resources`, so we don't have to keep modifying every POM whenever we use a plugin which can load config from the classpath.